ubuntu22.04/precompiled/Dockerfile

```diff
@@ -54,6 +54,11 @@ RUN apt-get update && apt-get install -y --download-only --no-install-recommends
     libnvidia-nscq-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 && \
     rm -rf /var/lib/apt/lists/*;
 
+RUN if [ "$DRIVER_BRANCH" -ge "550" ]; then \
+    apt-get update && \
+    apt-get install -y --no-install-recommends nvidia-imex-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 && \
```

Comment:
Shouldn't this be `download-only` like fabricmanager and libnvidia-nscq?

Reply:
For IMEX, I don't see any harm in always installing the package. Installing it at build time reduces the steps required at runtime and aligns with what we do for our non-precompiled containers. 

For fabric-manager / libnvidia-nscq, looks like we always download the packages, but only install them at runtime if we detect NVSwitch devices.